packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -2956,6 +2956,61 @@ export const getAccessRequirementAcl = (
   )
 }
 
+/**
+ * Delete the ACL for the access requirement with the given id.
+ * Only an ACT member is allowed to delete the ACL.
+ *
+ * See https://rest-docs.synapse.org/rest/DELETE/accessRequirement/requirementId/acl.html
+ */
+export const deleteAccessRequirementAcl = (
+  accessToken: string | undefined,
+  id: string | number,
+): Promise<void> => {
+  return doDelete(
+    ACCESS_REQUIREMENT_ACL(id),
+    accessToken,
+    BackendDestinationEnum.REPO_ENDPOINT,
+  )
+}
+
+/**
+ * Assign the given ACL to the access requirement with the given id.
+ * Only an ACT member is allowed to assign the ACL.
+ * Only supports REVIEW_SUBMISSIONS and EXEMPTION_ELIGIBLE access types.
+ *
+ * See https://rest-docs.synapse.org/rest/POST/accessRequirement/requirementId/acl.html
```

Comment:
Docs currently state that this only supports REVIEW_SUBMISSIONS, but are out of date -- Sandhra is planning to update the docs.